README.md

```diff
@@ -1,3 +1,17 @@
 # Validator kit
 
-TODO: add docs here how to use the validator kit
+## Overview
+
+This repository contains the validator kit for Mezo chain. The validator kit is a collection of tools and documentation to help you run a validator node on Mezo chain. There are three main components of the validator kit:
+
+1) Docker Compose - contains files to run a validator node using Docker. This is the recommended way to run a validator node as it requires less setup and maintenance.
+
+2) Native Binary - contains files to fetch the `mezod` binary from GCP Artifacts Registry and run it manually. This is an alternative way to run a validator node if you prefer to run it natively.
```

Comment:
Let's make it more generic and don't mention GCP AR. One day, this will be GH releases in `mezod` or maybe something else. Making it more generic will make this part future proof. At the same time, the information where the binary is stored is not crucial for validators if this is done automatically.
```suggestion
2) Native Binary - contains files to fetch the `mezod` binary from a remote repository and run it manually. This is an alternative way to run a validator node if you prefer to run it natively.
```